x/psiphon/psiphon_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package psiphon provides adaptors to use Psiphon as a StreamDialer.
+//
+// You will need to provide your own Psiphon config file, which you can get from the Psiphon team
+// or [generate one yourself].
+//
+// [generate one yourself]: https://github.com/Psiphon-Labs/psiphon-tunnel-core/tree/master?tab=readme-ov-file#generate-configuration-data
```

Comment:
This is duplicated. I'd suggest to remove it to keep only one single source of truth.

```suggestion
```